docs/content/install.sh

```diff
@@ -113,45 +113,48 @@ case $OS_type in
     ;;
 esac
 
+if [ "$OS" = 'osx' ] && [ "$OS_type" = 'arm' ]; then
```

Comment:
This is OK for the moment but for v1.55 we will support osx/arm

Reply:
Let's keep only the "fail on download errors" logic in the PR.
You can add the "arm -> amd64" trick on the stable branch directly.